RetailCoder.VBE/Properties/Settings.settings

```diff
@@ -0,0 +1,149 @@
+&lt;HotkeySetting xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"&gt;
+  &lt;Key1&gt;R&lt;/Key1&gt;
+  &lt;IsEnabled&gt;true&lt;/IsEnabled&gt;
+  &lt;HasShiftModifier&gt;true&lt;/HasShiftModifier&gt;
+  &lt;HasAltModifier&gt;false&lt;/HasAltModifier&gt;
+  &lt;HasCtrlModifier&gt;true&lt;/HasCtrlModifier&gt;
+  &lt;CommandTypeName&gt;CodePaneRefactorRenameCommand&lt;/CommandTypeName&gt;
+&lt;/HotkeySetting&gt;</Value>
+    </Setting>
+    <Setting Name="DefaultHotkey_RefactorEncapsulateFieldCommand" Type="Rubberduck.Settings.HotkeySetting" Scope="Application">
+      <Value Profile="(Default)">&lt;?xml version="1.0" encoding="utf-16"?&gt;
+&lt;HotkeySetting xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"&gt;
+  &lt;Key1&gt;F&lt;/Key1&gt;
+  &lt;IsEnabled&gt;true&lt;/IsEnabled&gt;
+  &lt;HasShiftModifier&gt;true&lt;/HasShiftModifier&gt;
+  &lt;HasAltModifier&gt;false&lt;/HasAltModifier&gt;
+  &lt;HasCtrlModifier&gt;true&lt;/HasCtrlModifier&gt;
+  &lt;CommandTypeName&gt;RefactorEncapsulateFieldCommand&lt;/CommandTypeName&gt;
+&lt;/HotkeySetting&gt;</Value>
+    </Setting>
+    <Setting Name="DefaultHotkey_RefactorExtractMethodCommand" Type="Rubberduck.Settings.HotkeySetting" Scope="Application">
+      <Value Profile="(Default)">&lt;?xml version="1.0" encoding="utf-16"?&gt;
+&lt;HotkeySetting xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"&gt;
+  &lt;Key1&gt;M&lt;/Key1&gt;
+  &lt;IsEnabled&gt;true&lt;/IsEnabled&gt;
+  &lt;HasShiftModifier&gt;true&lt;/HasShiftModifier&gt;
+  &lt;HasAltModifier&gt;false&lt;/HasAltModifier&gt;
+  &lt;HasCtrlModifier&gt;true&lt;/HasCtrlModifier&gt;
+  &lt;CommandTypeName&gt;RefactorExtractMethodCommand&lt;/CommandTypeName&gt;
+&lt;/HotkeySetting&gt;</Value>
+    </Setting>
+    <Setting Name="DefaultHotkey_RefactorMoveCloserToUsageCommand" Type="Rubberduck.Settings.HotkeySetting" Scope="Application">
+      <Value Profile="(Default)">&lt;?xml version="1.0" encoding="utf-16"?&gt;
+&lt;HotkeySetting xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"&gt;
+  &lt;Key1&gt;C&lt;/Key1&gt;
+  &lt;IsEnabled&gt;true&lt;/IsEnabled&gt;
+  &lt;HasShiftModifier&gt;true&lt;/HasShiftModifier&gt;
+  &lt;HasAltModifier&gt;false&lt;/HasAltModifier&gt;
+  &lt;HasCtrlModifier&gt;true&lt;/HasCtrlModifier&gt;
+  &lt;CommandTypeName&gt;RefactorMoveCloserToUsageCommand&lt;/CommandTypeName&gt;
+&lt;/HotkeySetting&gt;</Value>
+    </Setting>
+    <Setting Name="DefaultHotkey_CodeExplorerCommand" Type="Rubberduck.Settings.HotkeySetting" Scope="Application">
+      <Value Profile="(Default)">&lt;?xml version="1.0" encoding="utf-16"?&gt;
+&lt;HotkeySetting xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"&gt;
+  &lt;Key1&gt;C&lt;/Key1&gt;
```

Comment:
Missed that in code review. Ctrl+C (Copy) is not a shortcut we want to override.

Reply:
the fix is to set `HasAltModifier` to true, I think

Reply:
It used to be `Ctrl` + `R` (which overrides the VBE's shortcut for activating the Project Explorer.)

Reply:
@Vogel612 no, that hotkey was Ctrl+R *specifically* to replace the VBE's Project Explorer by default.